<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>АРМ Официанта</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{background: rgb(102, 102, 153)}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>270</width>
      <height>530</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	background: #471E70;
	border: 1px solid;
	border-radius:7px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>270</width>
       <height>490</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>270</width>
       <height>490</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>270</width>
       <height>530</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Anonymous Pro</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget{
	border: none;
	background:transparent;
	color: white;
}

QHeaderView{
	border: none;
	border-radius: 0;
}
QHeaderView::section{
	border: none;
	background:transparent;
	color: black;
}</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>0</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>70</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>№</string>
      </property>
      <property name="font">
       <font>
        <family>Anonymous Pro</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignTop</set>
      </property>
      <property name="background">
       <color>
        <red>71</red>
        <green>30</green>
        <blue>112</blue>
       </color>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Наименование</string>
      </property>
      <property name="font">
       <font>
        <family>Anonymous Pro</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignTop</set>
      </property>
      <property name="background">
       <color alpha="0">
        <red>71</red>
        <green>30</green>
        <blue>112</blue>
       </color>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
     </column>
     <column>
      <property name="text">
       <string>кол-во</string>
      </property>
      <property name="font">
       <font>
        <family>Anonymous Pro</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignTop</set>
      </property>
      <property name="background">
       <color alpha="0">
        <red>71</red>
        <green>30</green>
        <blue>112</blue>
       </color>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
     </column>
     <column>
      <property name="text">
       <string>итог</string>
      </property>
      <property name="font">
       <font>
        <family>Anonymous Pro</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignTop</set>
      </property>
      <property name="background">
       <color alpha="0">
        <red>71</red>
        <green>30</green>
        <blue>112</blue>
       </color>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-1</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Anonymous Pro</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	color: white;
	border: none;
	border-radius: 7px;
	background: transparent;
}</string>
     </property>
     <property name="text">
      <string> № Наименование  к-во   сумма</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>250</width>
       <height>2</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">Line{
	border: none;
	border-radius: 0;
	background-color: white;
	opacity: 0.5;
}</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>100</width>
      <height>580</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	background-color: #333333;
	border-radius: 7px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="tipsButton">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>12</y>
       <width>90</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Anonymous Pro</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	color: white;
	background-color: #000033;
	border: 1px solid black;
	border-radius: 7px;
}

QPushButton:hover{
	background-color: rgb(92, 100, 150);
}

QPushButton:pressed{
	background-color:rgb(0, 0, 30);
	border: 3px inset
}</string>
     </property>
     <property name="text">
      <string>Чаевые</string>
     </property>
    </widget>
    <widget class="QPushButton" name="payOptionMethod">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>67</y>
       <width>90</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Anonymous Pro</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	color: white;
	background-color: #000033;
	border: 1px solid black;
	border-radius: 7px;
}

QPushButton:hover{
	background-color: rgb(92, 100, 150);
}

QPushButton:pressed{
	background-color:rgb(0, 0, 30);
	border: 3px inset
}</string>
     </property>
     <property name="text">
      <string>Оплата:
наличными</string>
     </property>
    </widget>
    <widget class="QPushButton" name="payOptionTime">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>122</y>
       <width>90</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Anonymous Pro</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	color: white;
	background-color: #000033;
	border: 1px solid black;
	border-radius: 7px;
}

QPushButton:hover{
	background-color: rgb(92, 100, 150);
}

QPushButton:pressed{
	background-color:rgb(0, 0, 30);
	border: 3px inset
}</string>
     </property>
     <property name="text">
      <string>Оплата:
сразу</string>
     </property>
    </widget>
    <widget class="QPushButton" name="needsButton">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>177</y>
       <width>90</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Anonymous Pro</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	color: white;
	background-color: #000033;
	border: 1px solid black;
	border-radius: 7px;
}

QPushButton:hover{
	background-color: rgb(92, 100, 150);
}

QPushButton:pressed{
	background-color:rgb(0, 0, 30);
	border: 3px inset
}</string>
     </property>
     <property name="text">
      <string>Пожелания
к заказу</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>232</y>
       <width>90</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Anonymous Pro</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	color: white;
	background-color: #000033;
	border: 1px solid black;
	border-radius: 7px;
}

QPushButton:hover{
	background-color: rgb(92, 100, 150);
}

QPushButton:pressed{
	background-color:rgb(0, 0, 30);
	border: 3px inset
}</string>
     </property>
     <property name="text">
      <string>Постоянным
клиентам</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>20</y>
      <width>400</width>
      <height>580</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	background-color: #333333;
	border-radius: 7px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="soupButton">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>10</y>
       <width>190</width>
       <height>190</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Playfair Display</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: white;
    padding: -25px 0 0 0;
    border: 1px solid black;
    border-radius: 2px;
    background-position: center bottom;
    background-repeat: no-repeat;
    background-origin: content;
    background-image: url(&quot;:/qresource/images/img3.png&quot;);
 }</string>
     </property>
     <property name="text">
      <string>Супы</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>190</width>
       <height>189</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="secondButton">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>10</y>
       <width>190</width>
       <height>190</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Playfair Display</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: white;
    padding: -25px 0 0 0;
    border: 1px solid black;
    border-radius: 2px;
    background-position: center bottom;
    background-repeat: no-repeat;
    background-origin: content;
    background-image: url(&quot;:/qresource/images/img2.png&quot;);
 }</string>
     </property>
     <property name="text">
      <string>Вторые
блюда</string>
     </property>
     <property name="icon">
      <iconset theme="soups">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>190</width>
       <height>190</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="traditionalButton">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>210</y>
       <width>190</width>
       <height>190</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Playfair Display</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: white;
    padding: -25px 0 0 0;
    border: 1px solid black;
    border-radius: 2px;
    background-position: center bottom;
    background-repeat: no-repeat;
    background-origin: content;
    background-image: url(&quot;:/qresource/images/img5.png&quot;);
 }</string>
     </property>
     <property name="text">
      <string>Традиционные</string>
     </property>
     <property name="icon">
      <iconset theme="soups">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>190</width>
       <height>190</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="topDishesButton">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>210</y>
       <width>190</width>
       <height>190</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Playfair Display</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: white;
    padding: -25px 0 0 0;
    border: 1px solid black;
    border-radius: 2px;
    background-position: center bottom;
    background-repeat: no-repeat;
    background-origin: content;
    background-image: url(&quot;:/qresource/images/img4.png&quot;);
 }</string>
     </property>
     <property name="text">
      <string>Блюда
месяца</string>
     </property>
     <property name="icon">
      <iconset theme="soups">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>190</width>
       <height>190</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="customButton">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>410</y>
       <width>190</width>
       <height>190</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Playfair Display</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: white;
    padding: -25px 0 0 0;
    border: 1px solid black;
    border-radius: 2px;
    background-position: center bottom;
    background-repeat: no-repeat;
    background-origin: content;
    background-image: url(&quot;:/qresource/images/img1.png&quot;);
 }</string>
     </property>
     <property name="text">
      <string>Свой
рецепт</string>
     </property>
     <property name="icon">
      <iconset theme="soups">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>190</width>
       <height>190</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="sweetButton">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>410</y>
       <width>190</width>
       <height>190</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Playfair Display</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: white;
    padding: -25px 0 0 0;
    border: 1px solid black;
    border-radius: 2px;
    background-position: center bottom;
    background-repeat: no-repeat;
    background-origin: content;
    background-image: url(&quot;:/qresource/images/img6.png&quot;);
 }</string>
     </property>
     <property name="text">
      <string>Десерты</string>
     </property>
     <property name="icon">
      <iconset theme="soups">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>190</width>
       <height>190</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>270</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	background-color: #471E70;
	border-radius: 7px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="TotalLabel">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>570</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Anonymous Pro</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	border: none;
	color: white;
}</string>
    </property>
    <property name="text">
     <string>Итог:</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalValue">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>570</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Anonymous Pro</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	border: none;
	color: white;
}</string>
    </property>
    <property name="text">
     <string>0000000.00</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
